src/quo2/components/settings/category/view.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.components.settings.category.view
+  (:require
+    [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [quo2.components.settings.category.style :as style]))
+
+(defn category
```

Comment:
can you use the new theming approach here? :)

Reply:
@J-Son89 done :)